deployment-wildfly.md

```diff
@@ -1,15 +1,15 @@
 # Wildfly Deployment
 
 ## Wildfly version
-Note that these examples are tested with wildfly `13.0.0.Final`.
+Note that these examples are tested with wildfly `CASUAL_VERSION` `31.0.1.Final`.
```

Comment:
CASUAL_VERSION is a symbol that is set and used later on down below in cli script snippets.
So next time we can just adjust that instead of having the value in each place.